src/main/java/org/broadinstitute/hellbender/utils/smithwaterman/SmithWatermanIntelAligner.java

```diff
@@ -10,14 +10,11 @@
 
 public final class SmithWatermanIntelAligner implements SmithWatermanAligner {
```

Comment:
Needs class level javadoc explaining what this is, i.e. that it's an implementation of smith waterman aligner that's takes advantage of intel hardware optimizations.